src/TokenManagers/XboxTokenManager.js

```diff
@@ -60,11 +60,8 @@ class XboxTokenManager {
     // Because we do the full auth sequence like the official XAL, the issue with accounts under 18 (2148916238)
     // should not happen through title auth. But the user must always have an xbox.com profile before being able
     // to obtain a Minecraft or Xbox token.
-    switch (errorCode) {
-      case 2148916233: throw new URIError('Failed to obtain XSTS token: The user does not currently have an Xbox profile - https://signup.live.com/signup')
-      case 2148916238: throw new URIError('Failed to obtain XSTS token: The account date of birth is under 18 years and cannot proceed unless the account is added to a Family by an adult - https://account.microsoft.com/family')
-      default: throw new Error(`Failed to obtain XSTS token: ${JSON.stringify(response)}`)
-    }
+    if (Object.prototype.hasOwnProperty.call(xboxLiveErrors, errorCode)) throw new Error(xboxLiveErrors[errorCode])
```

Comment:
Can you elaborate what's going here ?

Reply:
StandardJS does not like you calling xboxLiveErrors.hasOwnProperty to see if the property exists. You have to call the prototype function to check. If it does, just output the message of the property to errors

Reply:
Yes because that's deprecated. If you need to see if a key exists in an object you can either do a simple get on it or use the in operator 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in